README.md

```diff
@@ -27,53 +22,196 @@ ember install ember-a11y-testing
 
 ## Usage
 
+Setting up `ember-a11y-testing` in your tests can be done in one of two ways:
+
+1. A one-time setup in your tests/test-helper.js file using `setupGlobalA11yHooks`
+1. In individual tests using the `a11yAudit` test helper.
+
 ### aXe Options
 
 When using the `a11yAudit` helper, you can pass in `axe-core` options.
 These options are documented in the [axe-core API docs](https://www.deque.com/axe/axe-for-web/documentation/api-documentation/#options-parameter).
 The rule definitions are documented on [dequeuniversity.com/rules](https://dequeuniversity.com/rules/axe/3.2).
 
-### Testing Usage
+Each of the following sections individually details how to set aXe options for your tests.
+
+### `setupGlobalA11yHooks` Usage
+
+The `setupGlobalA11yHooks` function is intended to be imported and invoked a single time for your entire test suite.
```

Comment:
Maybe mention here in the prose that it should probably be in `tests/test-helper.js`?